app/controllers/api/v1/profiles_controller.rb

```diff
@@ -0,0 +1,22 @@
+module Api
+  module V1
+    class ProfilesController < ApplicationController
+
+      before_action :doorkeeper_authorize!
+
+      respond_to :html, :json
+
+      def me
+        respond_with current_resource_owner
+        # render nothing: true
+        # render json: 'hello'
+      end
+
+      protected
+
+      def current_resource_owner
+        @current_resource_owner ||= User.find(doorkeeper_token.resource_owner_id) if doorkeeper_token
```

Comment:
Really really strange functionality, where do you use  `@current_resource_owner`, better to use '.find_by', what happened if doorkeeper_token doesn't exist?